tests/unit/type/zset.tcl

```diff
@@ -780,6 +780,44 @@ start_server {tags {"zset"}} {
             set oldscore $score
         }
     }
+    
+    test {ZMSCORE retrieve} {
+        r del zmscoretest
+        r zadd zmscoretest 10 x
+        r zadd zmscoretest 20 y
+        
+        r zmscore zmscoretest x y
+    } {10 20}
+
+    test {ZMSCORE retrieve from empty set} {
+        r del zmscoretest
+        
+        r zmscore zmscoretest x y
+    } {{} {}}
```

Comment:
I amended the PR to make fetching the zmscore for an empty zset return a list of nil values.

This was done to be consistent with https://redis.io/commands/hmget and zscore, which returns a single nil value for a missing sorted set

> Because non-existing keys are treated as empty hashes, running HMGET against a non-existing key will return a list of nil values.